accounts/tests.py

```diff
@@ -1,3 +1,29 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User
+from .models import UserAccount
 
-# Create your tests here.
+
+@pytest.fixture
+def test_account(db):
+    """test account to reduce redundancy"""
+    username = 'testuser'
+    password = 'lvkmdgv32fdbf1b'
+    email = 'testuser@test.com'
+    new_user = User.objects.create_user(
+        username=username, password=password, email=email
+    )
+    return new_user
+
+
+@pytest.mark.django_db()
+def test_user_create(test_account):
+    """Create a user and check if a UserAccount instance was created"""
+    assert User.objects.count() == 1
```

Comment:
@bardielle is it okay to compare the number of results? I think you said not to do it

Reply:
Changed it to UserAccount.objects.get(user=user_account). If it wasn't created, the test wouldn't pass anyway, so I think it works? Correct me if I'm wrong though.

Reply:
so you don't need that line

Reply:
The line that compares them or what I changed it to?  Should I remove that too?

Reply:
you added line 22, so line 21 is not needed

Reply:
Then what way would be better if I want to test the model inheritance? Or do I not need a test for that at all?